src/context/AppContextTypes.ts

```diff
@@ -0,0 +1,12 @@
+export enum MenuOptions {
+  HOME = 'home',
+  TRANSACTION = 'transaction',
+  ADDBUTTON = 'addButton',
+  BUDGET = 'budget',
+  PROFILE = 'profile',
+}
+
+export interface AppContextType {
+  activeButton: MenuOptions | null;
```

Comment:
Let's change naming in the AppContext:
- activeButton -> activeTab
- setActiveButton - setActiveTab
You'd need to update AppContext.tsx file (and other files where setActiveButton was used, for example)